mmv1/third_party/terraform/utils/node_config.go.erb

```diff
@@ -102,6 +102,30 @@ func schemaNodeConfig() *schema.Schema {
 								ForceNew:         true,
 								Description: `Size of partitions to create on the GPU. Valid values are described in the NVIDIA mig user guide (https://docs.nvidia.com/datacenter/tesla/mig-user-guide/#partitioning)`,
 							},
+							"gpu_sharing_config": &schema.Schema{
+								Type:         schema.TypeList,
+								MaxItems:     1,
+								Optional:     true,
+								ForceNew:     true,
+								ConfigMode: schema.SchemaConfigModeAttr,
+								Description:  `The configuration for GPU sharing options.`,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"gpu_sharing_strategy": &schema.Schema{
+											Type:        schema.TypeString,
+											Required:    true,
+											ForceNew:    true,
+											Description: `The type of GPU sharing strategy to enable on the GPU node. Possible value is TIME_SHARING`,
```

Comment:
Can we link to possible values instead? This documentation is likely to get outdated. markdown syntax is supported.

Reply:
done